core/src/components/button/button.ionic.scss

```
@@ -102,44 +97,31 @@
 // Button Sizes
 // -------------------------------------------------------------------------------
 
-/* Extra Small Button */
-:host(.button-xsmall) {
-  --padding-top: #{globals.$ion-space-100};
-  --padding-end: #{globals.$ion-space-300};
+/* Small Button */
+:host(.button-small) {
+  --padding-end: #{globals.$ion-space-400};
 
-  min-height: globals.$ion-space-600;
+  @include globals.typography(globals.$ion-body-action-sm);
 
-  font-size: globals.$ion-font-size-300;
+  min-height: #{globals.$ion-scale-1000};
 }
 
-/* Small Button */
-:host(.button-small) {
-  --padding-top: #{globals.$ion-space-200};
-  --padding-end: #{globals.$ion-space-400};
+/* Medium Button */
+:host(.button-medium) {
+  --padding-end: #{globals.$ion-space-500};
 
-  min-height: #{globals.$ion-scale-800};
+  @include globals.typography(globals.$ion-body-action-md);
 
-  font-size: #{globals.$ion-font-size-300};
+  min-height: #{globals.$ion-scale-1200};
 }
 
 /* Large Button */
 :host(.button-large) {
-  --padding-top: #{globals.$ion-space-400};
   --padding-end: #{globals.$ion-space-600};
```

```suggestion
  --padding-end: #{globals.$ion-space-700};
```

My request to change this was kind of hidden by the request to remove `--padding-top` but this also appears as `700` in Figma for me